### ISSUE TITLE
`mail_debug` incompatible with Python 3.12

### ISSUE THREAD

**Issue by michel-slm** [django-extensions/django-extensions#1831]
Python 3.12 dropped `asyncore` which has been deprecated since 3.6, recommending `asyncio` instead as the replacement:

https://docs.python.org/3.12/whatsnew/3.12.html#removed

**Comment by weslord** [comment]
Also `smtpd`

**Comment by bgtron** [comment]
It would be wonderful to get this resolved.  Just wanting to _bump_ this issue...

**Comment by sundayz-hunter** [comment]
I've just updated to 4.0 released yesterday.
The error persists: it's no longer `asyncore `but `aiosmtpd` that's causing the problem.

`Error fetching command 'mail_debug': Please install 'aiosmtpd' library to use mail_debug command.`

**Comment by browniebroke** [comment]
Correct, the command has been rewritten and now requires an 3rd party library to be installed. It's documented here: https://django-extensions.readthedocs.io/en/latest/mail_debug.html

Rationale as to why it's no installed by default: https://github.com/django-extensions/django-extensions/issues/1923
